test/integration/pkg/generated/openapi_generated.go

```diff
@@ -315,7 +336,7 @@ func schema_test_integration_testdata_enumtype_FruitsBasket(ref common.Reference
 				Properties: map[string]spec.Schema{
 					"content": {
 						SchemaProps: spec.SchemaProps{
-							Description: "\n\n\nPossible enum values:\n - `\"apple\"` is the Apple\n - `\"banana\"` is the Banana\n - `\"onigiri\"` is the Rice ball that does not seem to belong to a fruits basket but has a long comment that is so long that it spans multiple lines",
+							Description: "Possible enum values:\n - `\"apple\"` is the Apple\n - `\"banana\"` is the Banana\n - `\"onigiri\"` is the Rice ball that does not seem to belong to a fruits basket but has a long comment that is so long that it spans multiple lines",
```

Comment:
How can we have this here?

Reply:
I understand that, my concern is that the test should have failed in #286 but didn't

Reply:
The reason is that if the integration tests fail, the expected results get updated on disk instead of reporting as an error, which is bad.